### ISSUE TITLE
Error installing tweaks on Ubuntu (KeyError)

### ISSUE THREAD

**Issue by TheNerd1584** [Jonius7/SteamUI-OldGlory#75]
Error while installing tweaks.
~~~~~~~~~~~~~~~~~~~~
Traceback (most recent call last):
  File "/home/nerdo/Downloads/SteamUI-OldGlory-master/manager.py", line 51, in install_click
    settings = set_js_config(controller, settings)
  File "/home/nerdo/Downloads/SteamUI-OldGlory-master/manager.py", line 119, in set_js_config
    controller.frames["PageTwo"].js_gui.checkvars[js_name].set(settings[setting]["value"])
KeyError: 'Landscape Images JS Tweaks (beta, working, some layout quirks with shelves)'
~~~~~~~~~~~~~~~~~~~~

I'm on Ubuntu 22.10 and just installed old glory

**Comment by Jonius7** [comment]
This is a lingering issue that has popped up before #48
I've been unable to replicate it but could use a rewrite to fix this issue once and for all.

**Comment by Jonius7** [comment]
Should be fixed in [5.10.7](https://github.com/Jonius7/SteamUI-OldGlory/releases/tag/Release_5.10.7)

**Comment by TheNerd1584** [comment]
Yes the problem has been fixed
